tracking_manager_domain/models/models.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
```

Comment:
I suppose you have to change this copyright.

Reply:
Thank you - done.